src/lib/Field/ContentFieldResolverInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzRecommendationClient\Field;
+
+use eZ\Publish\API\Repository\Values\Content\Content;
+
+interface ContentFieldResolverInterface
+{
+    /**
+     * @return array<string, string|float|array>
```

Comment:
What's the accepted values that come from `Field->value`? I think that at the very least booleans are also acceptable? Anything actually that is not an object?

@adamwojs @alongosz ?

Reply:
Changed phpdoc: https://github.com/ezsystems/ezrecommendation-client/pull/61/files#diff-d88523d53922cb6b5ca291897726289ebb0555ac7e9eb0aa3415a42e8d8d2564R16

Get string value from `\eZ\Publish\SPI\FieldType\Value` https://github.com/ezsystems/ezrecommendation-client/pull/61/files#diff-932b5df64d7aaa907c2f57f97e7136e84e65a626e8cb2017fb37ae6ec27488f3R20